pages/administration/tools/booking-lookup.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <AdminPage title="Bookings Lookup">
+    <div class="flex items-end space-x-4">
+      <div>
+        <label>User</label>
+        <UiInputText
+            v-model="userSearch"
+                     placeholder="Filter by user" />
+      </div>
+      <div>
+        <!-- Show the productions filter if we have any. If not, disable the
+        filter.-->
+        <label>Production</label>
+        <UiInputSelect
+            v-if="!$apollo.queries.productions.loading && productions.length
+            > 0"
+            v-model="productionSlug"
+            nullify-value="performanceId"
+            :options="productions"
+        />
+        <UiInputSelect
+            v-else
+            v-model="productionSlug"
+            nullify-value="performanceId"
+            :disabled="true"
+            :options="[
+            { displayText: 'All', value: null }]"
+        />
+      </div>
+      <div>
+        <!-- Show the performances filter if we have any for a selected
+        production. If not, disable the filter.-->
+        <label>Performance</label>
+        <UiInputSelect
+            v-if="productionSlug && !$apollo.queries.performances.loading &&
+          performances.length > 0 && !disablePerformanceDropdown"
+            v-model="performanceId"
+            :options="performances"
+        />
+        <UiInputSelect
+            v-else
+            v-model="performanceId"
+            :disabled="true"
+            :options="[
+            { displayText: 'All', value: null }]"
+        />
+      </div>
+      <div>
+        <label>Status</label>
+        <UiInputSelect
+            v-model="bookingsStatus"
+            :options="[
+            { displayText: 'All', value: null },
+            { displayText: 'In Progress', value: 'IN_PROGRESS' },
+            { displayText: 'Paid', value: 'PAID' }
+          ]"
+        />
```

Comment:
Within the Django Admin panel's [StatusFilter](https://github.com/BristolSTA/uobtheatre-api/blob/dev/uobtheatre/bookings/admin.py?plain=1#L33-L57), there are currently three other statuses that are filterable: `Refunded`, `Locked` and `Cancelled`. If our end goal is to move away from needing to examing things through the admin panel, it'd be good to be able to search by those filters too.